tests/store/test_local_artifact_repo.py

```diff
@@ -1,101 +1,138 @@
 import os
-import unittest
+import pytest
 
 from mlflow.exceptions import MlflowException
 from mlflow.store.artifact_repository_registry import get_artifact_repository
```

Comment:
er, lint does not like this unimported error -- after lint passes i'm good with merging. thanks!

Reply:
Thanks!